package.json

```diff
@@ -6,37 +6,36 @@
   "types": "lib/index.d.ts",
   "scripts": {
     "build": "tsc",
-    "test": "npm run reseedTestEnv && jest --config jestconfig.json",
+    "test": "jest --config jestconfig.json",
     "format": "prettier --write \"src/**/*.ts\" \"src/**/*.js\"",
     "lint": "tslint -p tsconfig.json",
     "prepare": "npm run build",
     "prepublishOnly": "npm test && npm run lint",
     "preversion": "npm run lint",
     "version": "npm run format && git add -A src",
     "postversion": "git push && git push --tags",
-    "setupTestEnv": "cd test/local-dynamo && sls dynamodb install",
-    "startDynamo": "cd test/local-dynamo && sls dynamodb start",
-    "reseedTestEnv": "cd test/local-dynamo && sls dynamodb seed --seed=test"
+    "setupTestEnv": "true",
+    "startDynamo": "true",
+    "reseedTestEnv": "true"
```

Comment:
Thanks for the MR! What does the "true" here do?

Reply:
> Thanks for the MR! What does the "true" here do?

For compatibility reasons: `true` is a command in your system where it does nothing but exit with the status `0`.

If any external contributors have their own test setup, this change will ensure their setup remains working.